packages/grid/_modules_/grid/components/cell/GridRowCells.tsx

```diff
@@ -72,32 +74,40 @@ export const GridRowCells = React.memo((props: RowCellsProps) => {
 
     const cellParams: GridCellParams = apiRef!.current.getCellParams(id, column.field);
 
-    let cssClassProp = { cssClass: '' };
+    const classNames = [cellClassName];
+
     if (column.cellClassName) {
-      if (!isFunction(column.cellClassName)) {
-        cssClassProp = { cssClass: clsx(column.cellClassName) };
-      } else {
-        cssClassProp = { cssClass: column.cellClassName(cellParams) as string };
-      }
+      classNames.push(
+        clsx(
+          isFunction(column.cellClassName)
+            ? column.cellClassName(cellParams)
+            : column.cellClassName,
+        ),
+      );
     }
 
+    // TODO to be removed by https://github.com/mui-org/material-ui-x/issues/275
```

Comment:
I thought we would do this in this PR. 
We might end up with 3 ways to do this in a version if we do it separately at a later stage.

Reply:
Priorities might change.

Reply:
I didn't it here to separate things. This PR will not go into this release.